ui/components/Filters.js

```diff
@@ -165,6 +166,43 @@ function MesheryFilters({
   const DEPLOY_URL = '/api/filter/deploy';
 
 
+  const getMuiTheme = () => createTheme({
+    overrides : {
+      MuiInput : {
+        underline : {
+          "&:hover:not(.Mui-disabled):before" : {
+            borderBottom : "2px solid #222"
+          },
+          "&:after" : {
+            borderBottom : "2px solid #222"
+          }
+        }
+      },
+      MUIDataTableSearch : {
+        searchIcon : {
+          color : "#607d8b" ,
```

Comment:
@Tanuj22 we have this color in the central theme, right?

Reply:
@leecalcote I used the same central(#codes) that we have all across.